samples/apps/copilot-chat-app/webapp/src/libs/services/BaseService.ts

```diff
@@ -28,9 +28,14 @@ export class BaseService {
             'Content-Type': 'application/json',
         });
 
-        // For each enabled plugin, pass its auth information as a customer header
-        // to the backend so the server can authenticate to the plugin
+        // API key auth for private hosted instances
+        if (process.env.REACT_APP_SK_API_KEY) {
+            headers.append(`x-sk-copilot-api-key`, process.env.REACT_APP_SK_API_KEY as string);
```

Comment:
```suggestion
            headers.append(`x-sk-api-key`, process.env.REACT_APP_SK_API_KEY as string);
```

Reply:
Was literally in process of pushing the change haha